Classes/Command/PluginCommand.php

```diff
@@ -82,7 +82,10 @@ private function askForPluginInformation(SymfonyStyle $io, InputInterface $input
         );
 
         $referencedControllerActions = [];
+        $isTypoScriptCreation = false;
+        $set = null;
```

Comment:
Please use more describing variable names. What do you want to "set" here?

Reply:
The set to be used, one of Configutation/Sets, how would you call them?